CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changelog
 =========
 
+ * **2017-02-17**: [BC BREAK] DynamicRouter::setRequest changed to DynamicRouter::setRequestStack for better DI.
```

Comment:
"[BC BREAK] `DynamicRouter::setRequest()` has been removed in favor of `DynamicRouter::setRequestStack()`."

I would not mention "for better DI", it sounds like a bit strange reason.